modules/assim.sequential/NAMESPACE

```diff
@@ -9,7 +9,11 @@ export(EnKF)
 export(EnKF.MultiSite)
 export(GEF)
 export(Local.support)
+export(Obs.data.prepare.MultiSite)
+export(Remote_Sync_launcher)
```

Comment:
curious, why do some functions have "." and others have "_"?

Reply:
@para2x bump.....I think we came up with a style guide for this, correct @ashiklom @tonygardella ?

Reply:
If its a super pain to change, maybe we can make a SDA cleanup issue to tackle this after the Hackathon?